docs/src/CommutativeAlgebra/orderings.md

```diff
@@ -53,22 +56,58 @@ Here are some illustrating examples:
 
 ##### Examples
 
-```@repl oscar
-S, (w, x) = PolynomialRing(QQ, ["w", "x"])
-o = lex([w, x])
-canonical_matrix(o)
-cmp(o, w^2*x, x^3)
-R, (w, x, y, z) = PolynomialRing(QQ, ["w", "x", "y", "z"])
-o1 = degrevlex([w, x])
-is_global(o1)
-canonical_matrix(o1)
-o2 = neglex([y, z])
-is_local(o2)
-canonical_matrix(o2)
-o3 = o1*o2
-canonical_matrix(o3)
-is_mixed(o3)
-show(collect(terms((1+w+x+y+z)^2, o3)))
+```jldoctest
+julia> S, (w, x) = PolynomialRing(QQ, ["w", "x"])
+(Multivariate Polynomial Ring in w, x over Rational Field, fmpq_mpoly[w, x])
+
+julia> o = lex([w, x])
+lex([w, x])
+
+julia> canonical_matrix(o)
+[1   0]
+[0   1]
+
+julia> cmp(o, w^2*x, x^3)
+1
+
+julia> R, (w, x, y, z) = PolynomialRing(QQ, ["w", "x", "y", "z"])
+(Multivariate Polynomial Ring in w, x, y, z over Rational Field, fmpq_mpoly[w, x, y, z])
+
+julia> o1 = degrevlex([w, x])
+degrevlex([w, x])
+
+julia> is_global(o1)
+true
+
+julia> canonical_matrix(o1)
+[1    1   0   0]
+[0   -1   0   0]
+
+julia> o2 = neglex([y, z])
+neglex([y, z])
+
+julia> is_local(o2)
+true
+
+julia> canonical_matrix(o2)
+[0   0   -1    0]
+[0   0    0   -1]
+
+julia> o3 = o1*o2
+degrevlex([w, x])*neglex([y, z])
+
+julia> canonical_matrix(o3)
+[1    1    0    0]
+[0   -1    0    0]
+[0    0   -1    0]
+[0    0    0   -1]
+
+julia> is_mixed(o3)
+true
+
+julia> show(collect(terms((1+w+x+y+z)^2, o3)))
+ERROR: MethodError: no method matching terms(::fmpq_mpoly, ::MonomialOrdering{FmpqMPolyRing})
```

Comment:
@tthsqe12 @wdecker is this error expected?

Reply:
Yes! Please just erase the above. We will discuss examples for terms with another PR at another part of the docu.